google/cloud/aiplatform/__init__.py

```diff
@@ -26,6 +26,7 @@
     TextDataset,
     VideoDataset,
 )
+from google.cloud.aiplatform.metadata import MetadataStore
```

Comment:
We shouldn't expose MetadataStore to the user. Only the underlying APIs.

Reply:
Agree. I was just exposing it to make our testing easier. Once we have the other plumbing in place, I was thinking of making it private? If I can call it from the notebook by being private, we can do the change now as well? WDYT?

Reply:
synced offline. Will make it private